<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bootloader</class>
 <widget class="QDialog" name="bootloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DFU Bootloader Interface</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xprotolabinterface.qrc">
    <normaloff>:/images/images/download2.png</normaloff>:/images/images/download2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupDeviceSelect">
     <property name="title">
      <string>Device Select</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioOW">
        <property name="toolTip">
         <string>Target: atxmega256a3u</string>
        </property>
        <property name="text">
         <string>Oscilloscope Watch</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXprotolab">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xprotolab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXminilab">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xminilab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXminilabB">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xminilab-2.4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXprotolabPortable">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xprotolab Portable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXminiulabPortable">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xminilab Portable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXminiulabPortable1_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xminilab Portable 1.11</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioXprotolabPlain">
        <property name="toolTip">
         <string>Target: atxmega32a4u</string>
        </property>
        <property name="text">
         <string>Xprotolab Plain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushAutoLoad">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Download corresponding files from www.gabotronics.com</string>
       </property>
       <property name="text">
        <string>Download Latest Firmware</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/download.png</normaloff>:/images/images/download.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Target</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTarget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushSelectFlashFile">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Manually select flash file</string>
       </property>
       <property name="text">
        <string>Select Flash File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Flash File</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineFlash"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushSelectEEPROMFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Manually select EEPROM file</string>
       </property>
       <property name="text">
        <string>Select EEPROM File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>EEPROM File</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEEPROM">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushReadFlash">
       <property name="text">
        <string>Read Flash</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/readtqfp.png</normaloff>:/images/images/readtqfp.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushReadEE">
       <property name="text">
        <string>Read EEPROM</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/readtqfp.png</normaloff>:/images/images/readtqfp.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushErase">
       <property name="text">
        <string>Erase Chip</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/BlankCheck.png</normaloff>:/images/images/BlankCheck.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushFlash">
       <property name="text">
        <string>Program Flash</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/writetqfp.png</normaloff>:/images/images/writetqfp.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushEEPROM">
       <property name="text">
        <string>Program EEPROM</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/writetqfp.png</normaloff>:/images/images/writetqfp.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="toolTip">
        <string>Exit bootloader</string>
       </property>
       <property name="text">
        <string>Start Application</string>
       </property>
       <property name="icon">
        <iconset resource="xprotolabinterface.qrc">
         <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkPreserveEE">
        <property name="text">
         <string>Preserve EEPROM</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushDoEverything">
        <property name="text">
         <string>Read EEPROM -&gt; Erase Chip -&gt; Program Flash -&gt; Program EEPROM -&gt; Start Application</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="xprotolabinterface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>bootloader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bootloader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
